examples/basic_operations.js

```diff
@@ -1,30 +1,47 @@
-"use strict";
-
-var Redis = require("ioredis");
-var redis = new Redis();
+const Redis = require("ioredis");
+const redis = new Redis({
+  port: process.env.redisPort,
+  host: process.env.redisEndpoint,
+  password: process.env.redisPW
+});
 
 // ioredis supports all Redis commands:
-redis.set("foo", "bar");
+redis.set("foo", "bar"); // returns promise which resolves to string, "OK"
```

Comment:
I'm considering putting all of this within an async function so that we can async/await some of the commands. What do you think?

Reply:
Sorry for the late response! Yes I think that would be helpful but still we should keep some examples for callback-style. I'm going to merge this pr so feel free to create a new pr for the proposal. Thanks again for the contribution!